GoogleForCreators/web-stories-wp
# Dashboard: Added animations to DIY template

## Summary

This PR adds animations to the DIY template.

## To-do

- ~I still need to update our missing animation parts spreadsheet to reflect the changes for DIY.~

## User-facing changes

None.

## Testing Instructions

0.) Enable animations by setting `enableAnimations` to `true` in `Dashboard.php`.
1.) Go to "Explore Templates".
2.) Click view on "DIY".
3.) Click on each page of the DIY template to see each one animate as they load.

---



Fixes #2206 #2872 

## Screenshots

Page 1:
<img src="https://user-images.githubusercontent.com/40646372/87363522-31040e80-c526-11ea-9c8f-7fd6da609d9f.gif" height="300">

Page 2:
<img src="https://user-images.githubusercontent.com/40646372/87363530-33feff00-c526-11ea-9d22-763cc5ce18a6.gif" height="300">

Page 3:
<img src="https://user-images.githubusercontent.com/40646372/87363534-36615900-c526-11ea-94e2-dedd0c2fccda.gif" height="300">

Page 4:
<img src="https://user-images.githubusercontent.com/40646372/87363537-382b1c80-c526-11ea-95a8-af448f6de723.gif" height="300">

Page 5:
<img src="https://user-images.githubusercontent.com/40646372/87363541-39f4e000-c526-11ea-94c2-f4f69ab232d2.gif" height="300">

Page 6:
<img src="https://user-images.githubusercontent.com/40646372/87363547-3bbea380-c526-11ea-8690-051a293ae4d7.gif" height="300">

Page 7:
<img src="https://user-images.githubusercontent.com/40646372/87363550-3e20fd80-c526-11ea-9c60-624f2ee6889d.gif" height="300">

Page 8:
<img src="https://user-images.githubusercontent.com/40646372/87363553-40835780-c526-11ea-88cc-54ce3020b90f.gif" height="300">

Page 9:
<img src="https://user-images.githubusercontent.com/40646372/87363556-424d1b00-c526-11ea-93cf-a44caddc0ff1.gif" height="300">

Page 10:
<img src="https://user-images.githubusercontent.com/40646372/87363560-4416de80-c526-11ea-9709-e57cdb18078a.gif" height="300">
